elyra/tests/metadata/test_metadata_app.py

```diff
@@ -1531,6 +1532,401 @@ def test_export_clean(script_runner, mock_data_dir):
     temp_dir.cleanup()
 
 
+def test_import_help(script_runner):
+    ret = script_runner.run("elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, "--help")
+    assert ret.success is False
+    assert f"\nImport metadata instances into schemaspace '{METADATA_TEST_SCHEMASPACE}'" in ret.stdout
+
+
+def test_import_no_directory(script_runner):
+    ret = script_runner.run("elyra-metadata", "import", METADATA_TEST_SCHEMASPACE)
+    assert ret.success is False
+    assert "ERROR: '--directory' is a required parameter." in ret.stdout
+
+
+def test_import_bad_argument(script_runner):
+    ret = script_runner.run(
+        "elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, "--directory=dummy-directory", "--bogus-argument"
+    )
+    assert ret.success is False
+    assert "ERROR: The following arguments were unexpected: ['--bogus-argument']" in ret.stdout
+
+
+def test_import_bad_schemaspace(script_runner):
+    ret = script_runner.run("elyra-metadata", "import", "bogus-schemaspace")
+    assert ret.success is False
+    assert "Subcommand 'bogus-schemaspace' is invalid." in ret.stdout
+
+
+def test_import_inaccessible_directory(script_runner):
+    directory_parameter = "/dummy-directory"
+
+    ret = script_runner.run(
+        "elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, "--directory={}".format(directory_parameter)
+    )
+    assert ret.success is False
+    assert (
+        f"Unable to reach the '{directory_parameter}'"
+        + f" directory: No such file or directory: '{directory_parameter}"
+        in ret.stdout
+    )
+
+
+def test_import_empty_directory(script_runner):
+    # create import directory
+    temp_dir = TemporaryDirectory()
+    directory_parameter = temp_dir.name
+
+    # import metadata
+    ret = script_runner.run(
+        "elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, "--directory={}".format(directory_parameter)
+    )
+    assert ret.success is True
+    assert f"No instances for import found in the '{directory_parameter}' directory" in ret.stdout
+    temp_dir.cleanup()
+
+
+def test_import_non_json_file(script_runner):
+    # create import directory
+    temp_dir = TemporaryDirectory()
+    directory_parameter = temp_dir.name
+
+    # add a dummy file in the directory
+    dummy_filename = "dummy.txt"
+    dummy_filepath = os.path.join(directory_parameter, dummy_filename)
+    dummy_file_content = "This is a dummy txt file."
+    f = open(dummy_filepath, "w")
+    f.write(dummy_file_content)
+    f.close()
+    assert os.path.exists(dummy_filepath)
+    assert open(dummy_filepath).read() == dummy_file_content
+
+    # import metadata
+    ret = script_runner.run(
+        "elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, "--directory={}".format(directory_parameter)
```

Comment:
We have switched to using f-strings only

```suggestion
        "elyra-metadata", "import", METADATA_TEST_SCHEMASPACE, f"--directory={directory_parameter}")
```